.ruby-version

```diff
@@ -0,0 +1 @@
+1.9.3-p551
```

Comment:
Ok cool. But I repeat my question - if the specs are the only code, why are we testing against multiple rubies? There's no reason, right? Or am I missing something?